docs/visual-basic/toc.yml

```diff
@@ -875,14 +875,17 @@
     - name: XML Processing Instruction Literal
       href: language-reference/xml-literals/xml-processing-instruction-literal.md
   - name: Error Messages
-    href: language-reference/error-messages/index.md
     items:
+    - name: Overview
+      href: language-reference/error-messages/index.md
+    - name: BC30663
```

Comment:
These are the only two that I replaced error messages with error IDs in the TOC so far.